docker_templates/templates/docker_images/create_ros_core_image.Dockerfile.em

```diff
@@ -41,7 +41,7 @@ RUN echo "deb http://packages.ros.org/ros/ubuntu `lsb_release -sc` main" > /etc/
 
 @(TEMPLATE(
     'snippet/install_ros_bootstrap_tools.Dockerfile.em',
-    ros_version='1',
+    ros_version=version,
```

Comment:
Again, is version not used elsewhere, such as pining the package version in the apt install of ros/gazebo packages?
https://github.com/osrf/docker_images/pull/244#discussion_r268011526